extractor/docker/docker_test.go

```diff
@@ -13,13 +17,30 @@ import (
 	"testing"
 	"time"
 
-	"github.com/aquasecurity/fanal/cache"
+	"github.com/klauspost/compress/zstd"
+
 	"github.com/aquasecurity/fanal/extractor"
 	"github.com/aquasecurity/fanal/types"
 	"github.com/docker/docker/client"
+	"github.com/genuinetools/reg/registry"
+	"github.com/opencontainers/go-digest"
+	bolt "github.com/simar7/gokv/bbolt"
+	kvtypes "github.com/simar7/gokv/types"
 	"github.com/stretchr/testify/assert"
 )
 
+func setupCache() (*bolt.Store, *os.File, error) {
+	f, err := ioutil.TempFile(".", "Bolt_TestStore-*")
+	if err != nil {
+		return nil, nil, err
+	}
+
+	s, err := bolt.NewStore(bolt.Options{
```

Comment:
In the point of unit test, we should prepare another Store like MemoryStore. 
This will be helpful. https://github.com/spf13/afero#step-2-declare-a-backend

Reply:
I'll add a comment for future improvement to this.